src/pages/MyKiva/MyKivaPage.vue

```diff
@@ -378,7 +376,24 @@ const fetchContentfulHeroData = () => {
 		});
 };
 
+onServerPrefetch(async () => {
+	await apollo.query({
```

Comment:
Does the hook await promises like expected? I know there are otherwise issues with waiting for promises in Vue 3, for example created doesn't wait.

Reply:
Yeah, works like expected, I tested both cases, just await promises is that we want, otherwise doesn't work prefetching apollo cache.

Reply:
Good approach for future pages on vue3...

Reply:
Thanks for verifying, it's structured as they instruct in the docs. I think it's essentially the replacement for created in vue3.

Reply:
Not precisely a replacement, if you have a vue3 environment without SSR support this doesn't work, basically from docs:
**Registers an async function to be resolved before the component instance is to be rendered on the server.**